agenthub/dummy_agent/agent.py

```diff
@@ -0,0 +1,21 @@
+"""Module for a Dummy agent."""
+
+from opendevin.action.base import NullAction
+from opendevin.state import State
+from opendevin.action import Action
+from typing import List
+from opendevin.agent import Agent
+from opendevin.controller.agent_controller import AgentController
+from opendevin.observation.base import NullObservation, Observation
+
+class DummyAgent(Agent):
```

Comment:
There's another PR that introduces a DummyAgent: https://github.com/OpenDevin/OpenDevin/pull/1316/files

Reply:
Hmm, OK, depending on which one gets merged in first we may have to dedup later.